tests/acceptance/helpers/codify.js

```diff
@@ -0,0 +1,29 @@
+const _ = require('lodash')
+const { passwords } = require('./userSettings')
+
+/**
+ * Gets the password for the un-initialized users and replaces the password accordingly
+ *
+ * @param {string} input
+ * @returns {string}
+ */
+exports.replaceInlineCode = function (input) {
+  const { client } = require('nightwatch-api')
+  const codes = {
+    ...passwords,
+    remote_backend_url: client.globals.remote_backend_url,
+    backend_url: client.globals.backend_url
+  }
+
+  const interpolate = /%([\s\S]+?)%/g
+  const compiled = _.template(input, { interpolate })
+  return compiled(codes)
+}
+
+exports.replaceInlineTable = function (dataTable) {
+  // using private variable of datatable, could change later
+  dataTable.rawTable = dataTable.rawTable.map(
+    rows => rows.map(exports.replaceInlineCode)
+  )
```

Comment:
This currently uses `.rawTable` private variable. We can make it use public api by following way:
```suggestion
dataTable.raw().forEach(
    row => row.forEach(
        (cell, index, arr) => arr[index] = replaceInlineCode(cell)
   )
);
```